src/udgl/menu/FloatingMenu.svelte

```diff
@@ -73,7 +77,8 @@ $: if (element && parent) {
   <div class=bound-menu bind:this={element} style="
     left: {leftPlacement}px;
     top: {topPlacement}px;
+    max-height: calc(100vh - {topPlacement}px - var(--space-6x));
```

Comment:
Nice!

Reply:
I would like to see the `var(--space-6x)` be codified somewhere else, maybe as `--screen-buffer` or something like that.

Reply:
or `--screen-padding`?